index.js

```diff
@@ -1,9 +1,9 @@
-module.exports = function reverse (a) {
-  var length = a.length
-  var buffer = new Buffer(length)
+module.exports = function reverse (src) {
+  var buffer = new Buffer(src.length)
 
-  for (var i = 0, j = length - 1; i < length; ++i, --j) {
-    buffer[i] = a[j]
+  for (var i = 0, j = src.length - 1; i <= j; ++i, --j) {
+    buffer[i] = src[j]
+    buffer[j] = src[i]
```

Comment:
Is this actually worth it?
I'd of though it'd create cache issues when iterating huge arrays (or even small ones?)